src/RpcClient/TransactionManager.cs

```diff
@@ -84,8 +84,6 @@ public TransactionManager MakeTransaction(byte[] script, TransactionAttribute[]
                 long remainder = Tx.SystemFee % d;
                 if (remainder > 0)
                     Tx.SystemFee += d - remainder;
```

Comment:
I think that now we can remove the `remainder`

Reply:
Done by #219